doc/guidelines/ui.md

```diff
@@ -0,0 +1,38 @@
+# UI
+
+## Submitting Data
+
+Data should be always submitted on-the-fly but with debouncing.
+
+## Okay Buttons
+
+If needed, the confirming button should be on the left.
+
+## Gender
+
+Use plural to avoid gender, avoid he/she.
+
+## Raster
+
+We raster per pixel, which should represent 1cm in reality.
+Geometric operations that produce results not in that raster, should be rounded.
+In the state or in the backend only integer values should be used.
+
+## Key bindings
+
+A JSON file is responsible for global key bindings.
+
+## Colors
+
+We use an uniform color theme for everything related to the UI.
```

Comment:
```suggestion
We use a uniform color theme for everything related to the UI.
```
Since uniform is pronounced with a consonant in the beginning ("juniform"), a is the correct indefinite article.
[Source](https://www.limlessons.com/en/blog_post/a-uniform-or-an-uniform-learn-to-use-indefinite-articles-in-english)